### ISSUE TITLE
Click on the film category on the Android TV display

### ISSUE THREAD

**Issue by joepermana94** [recloudstream/cloudstream#519]
### Describe your suggested feature

Can it be that in the next update the display of film categories on Android TV can be clicked like the display on a phone, so that you can choose more films ? Thankyou
![Screenshot_20230727-220444](https://github.com/recloudstream/cloudstream/assets/139651590/f8b8396a-9150-4c4b-b200-686a2ad297c3)
![Screenshot_20230727-220438](https://github.com/recloudstream/cloudstream/assets/139651590/80f1c054-9ee2-4541-94a0-6f93ab0181cd)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by dondeepak** [comment]
Yes this will be helpful because every time when we click on the any movie the list is reset to the starting position this makes explore movies hard on Android tv. If we have this feature it will be good
